src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,29 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import users from '../../api/users';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed } = todo;
+  const user = users.find(person => person.id === todo.userId) || null;
```

Comment:
Let's prepare the data in one place - in the App component.
The only way to pass some data in React using `props`.